types/parsimmon/parsimmon-tests.ts

```diff
@@ -324,6 +329,7 @@ numPar = P.digit
 
 fooArrPar = fooPar.sepBy(barPar);
 fooArrPar = fooPar.sepBy1(barPar);
```

Comment:
Just like `str1ArrPar`, can we remove this line if we already have `foo1ArrPar`?

Reply:
Same as above (verifies that `Parser<['foo', ...'foo'[]]>` is still recognized as a subtype of `Parser<'foo'[]>`); though the marginal improvement over plain string is obviously minor.